platform/nodejs/eslintrc.json

```diff
@@ -0,0 +1,18 @@
+{
+    "env": {
+        "es2021": true,
+        "node": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "rules": {
+        "indent": [ "warn", 4 ],
+        "getter-return": "off",
```

Comment:
Some rules have been turned off globally because there were too many errors and it's not clear if they should be fixed. Please remove the respective line to see the errors.